spec/models/manageiq/providers/ibm_power_hmc/infra_manager/event_parser_spec.rb

```diff
@@ -0,0 +1,144 @@
+describe ManageIQ::Providers::IbmPowerHmc::InfraManager::EventParser do
+  class TestEvent
```

Comment:
What data type are the underlying PowerHMC events raised as? Could you build one of those objects instead of defining a new class